codingTest_김준우/체육복 다른 풀이.js

```diff
@@ -0,0 +1,31 @@
+function solution(n, lost, reserve) {
+  let total = [];
+  let answer = 0;
+  for (let i = 0; i < n; i++) {
+    total[i] = 1;
+  }
+  lost.forEach((data) => (total[data - 1] -= 1));
+  reserve.forEach((data) => (total[data - 1] += 1));
```

Comment:
```suggestion
  // 체육복을 잃어버린 학생들의 체육복 수를 0으로
    lost.map((lost) => {allStudent[lost - 1] = 0});    
    // 여벌을 가지고 있는 학생들의 체육복 수 1 증가
    reserve.map((reserve) => {allStudent[reserve - 1] += 1});
```
같은 방법으로 map 메소드도 있어요